examples/oed_utils.jl

```diff
@@ -176,22 +176,22 @@ function add_to_min(x, u)
         if x[perm[i]] < u[perm[i]]
             x[perm[i]] += 1
             break
-        else
-            continue
         end
     end
     return x
 end
 
+"""
+We want to add to the smallest value of x while respecting the upper bounds u.
+In constrast to the function above, we do not require x to have zero entries.
```

Comment:
it can be impractical to refer to the function above in docstring, because the main use case is checking the help from the REPL.

Also:
> we do not require x to have zero entries

Replacement: we do not require `x` to have coordinates at zero

(to avoid the ambiguity on the meaning of having "zero entries"